src/index.html

```diff
@@ -5,9 +5,47 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="card__img"
+      >
+
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__reviews">
+        <div class="card__reviews-stars-4">
+          <div class="card__stars-star"></div>
+          <div class="card__stars-star"></div>
+          <div class="card__stars-star"></div>
+          <div class="card__stars-star"></div>
+          <div class="card__stars-star"></div>
+        </div>
+        <p class="card__review-tex">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <span class="price">Price:</span>
```

Comment:
```suggestion
        <span class="card__price-label">Price:</span>
```
follow the BEM class naming

Reply:
I think I understood what you meant. I seem to have corrected it correctly.